src/views/layout/mixin/ResizeHandler.ts

```diff
@@ -2,6 +2,7 @@ import { Component, Vue, Watch } from 'vue-property-decorator';
 import { DeviceType, AppModule } from '@/store/modules/app';
 
 const WIDTH = 992; // refer to Bootstrap's responsive design
+const RATIO = 3
```

Comment:
Please revert this file, I just changed this in the a commit, according to [this](https://github.com/PanJiaChen/vue-admin-template/commit/7725f72bebc51db48f444e22f71572d3e9ed7ee0).